src/Data/Aeson/V2/Key.hs

```diff
@@ -0,0 +1,53 @@
+{-# LANGUAGE CPP #-}
+
+module Data.Aeson.V2.Key
```

Comment:
This is mostly obvious but might be nice to say

```suggestion
-- | Follows the latest @aeson-2.*@ version of "Data.Aeson.Key" 
module Data.Aeson.V2.Key
```

Reply:
Yeah, that works. I try to describe how every `Data.Aeson.VN.X` is meant to relate to `aeson-N:Data.Aeson.X` in the README, but a header on each model is clearer and more easily discovered.